content/blog/2022/12/2022-12-05-jenkins-newsletter.adoc

```diff
@@ -88,9 +88,7 @@ Contributed by: https://github.com/dduportal[Damien Duportal]
 * JDK17 for Windows provided to developers.
 * https://meetings.jenkins-ci.org/[https://meetings.jenkins-ci.org/] website (archive Board meeting minutes) is recovered and back.
 * Deprecated "windows-slaves" plugin removed.
-* Azure networks are now code-managed to allow incoming IPv6 support and
-improved network performances.
-
+* Azure networks are now code-managed to allow incoming IPv6 support and improved network performances.
```

Comment:
That is work in progress and both the user visible results (ipv6 and improved network performance) are not there yet 😅
```suggestion
```